### PR TITLE
More Items Correction (In general Equips) And new item 1835

### PR DESCRIPTION
**Pull request by BrOgBr**
Most errors in Item's Script. _(but in some items, Weight and ATK/MATK rate too)_ 

**And new item "1835" Spartacus [2].**

I've used **DivinePride** _(Using All Servers to compare)_ and sometimes the iRO DB.

Special thanks to @sader1992 and Haruka Mayumi. _(helped with some script)._

Sorry for any mistake, I've never used this PR.

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -528,7 +528,7 @@
 1170,Katzbalger,Katzbalger,5,20,,2000,175,,1,0,0x00004082,63,2,34,4,48,1,3,{ bonus bVit,5; bonus bDef,10; },{},{}
 1171,Zweihander_,Zweihander,5,20,,2200,200,,1,2,0x00004082,63,2,34,4,48,1,3,{ bonus bUnbreakableWeapon,1; },{},{}
 1172,Claymore_,Claymore,5,74000,,2500,180,,1,2,0x00004080,63,2,34,3,33,1,3,{},{},{}
-1173,Muramasa_C,Muramasa,5,1,,0,204,,1,0,0x00004082,63,2,34,4,1,0,3,{ bonus bCritical,30; bonus bAspdRate,8; },{},{}
+1173,Muramasa_C,Muramasa,5,1,,0,204,,1,0,0x00004082,63,2,34,4,1,0,3,{ bonus bCritical,30; bonus bAspdRate,8; bonus2 bAddEff,Eff_Curse,10; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: This is iRO specific. If you look on divine-pride and switch server to kRO this effect is not mentioned in the item description.

File: db/re/item_db.txt [hunk 2]
```diff
@@ -679,7 +679,7 @@
 1314,F_Tomahawk_C,Tomahawk,5,2,,0,200,,1,0,0x000444A2,63,2,34,4,0,0,7,{ bonus bAtkEle,Ele_Wind; },{},{}
 1315,F_Right_Epsilon_C,Light Epsilon,5,1,,0,229,,1,0,0x000444A2,63,2,34,4,1,0,7,{ bonus bAtkEle,Ele_Holy; bonus bStr,10; },{},{}
 1316,Adventure_Axe,Adventure Axe,5,0,,0,100,,1,0,0x00040420,63,2,2,1,1,0,6,{},{},{}
-1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,1,1,1,6,{},{},{}
+1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,1,1,1,6,{ bonus bMaxHP,200; bonus bAspdRate,5; bonus bAspdRate,BaseLevel/10; bonus bMaxHP,(-40*(BaseLevel/10)); },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Missing the unbreakable bonus(`bonus bUnbreakableWeapon,1;`) and the required equip level(30).
It looks like the base level is only related to the aspd decrease, but that is just my interpretation of the korean translation.

File: db/re/item_db.txt [hunk 3]
```diff
@@ -1020,12 +1020,12 @@
 1668,Sword_Stick,Sword Stick,5,10,,500,120:150,,,2,0x800200,63,2,2,4,80,1,10,{ bonus bAspdRate,10; },{},{}
 1669,Thanos_Staff,Thanatos Staff,5,10,,1000,100:200,,1,1,0x00018314,56,2,2,4,120,1,10,{ bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus bHealPower,15; bonus bMagicHPGainValue,500; bonus bMagicSPGainValue,50; bonus2 bHPLossRate,100,10000; },{},{ heal -1000,0; }
 1670,RWC_Memory_Staff,RWC Memory Staff,5,20,,500,25:30,,1,1,0x00818315,63,2,2,3,1,1,10,{ .@r = getrefine(); bonus bMatk,30*(.@r/3); if(.@r>=6) bonus2 bMagicAddClass,Class_All,(.@r>=9?10:5); if(.@r>=9) bonus4 bAutoSpell,"HW_MAGICPOWER",1,10,0; },{},{}
-1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_Demon,10; bonus bMatkRate,10; },{},{}
+1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_Demon,10; .@r = getrefine(); if(.@r>=9) bonus bMatkRate,5; if(.@r>=12) bonus bMatkRate,7; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: I don't think these two bonus should stack. I think it's if >= 12 give 7% else if >= 9 give 5%.

File: db/re/item_db.txt [hunk 4]
```diff
@@ -1020,12 +1020,12 @@
...
 1673,Half_BF_Staff4,Half BF Staff4,5,20,,0,70:125,,1,0,0x00818314,63,2,2,3,80,1,10,{ bonus bInt,3; bonus bDex,2; bonus bDelayRate,-10; bonus2 bSPLossRate,5,12000; bonus bUnbreakableWeapon,1; },{},{}
 1674,Half_BF_Staff2,Half BF Staff2,5,20,,0,70:125,,1,0,0x00818314,63,2,2,3,80,1,10,{ bonus bInt,3; bonus bDex,3; bonus2 bMagicAddRace,RC_DemiHuman,10; bonus2 bMagicAddRace,RC_Player,10; bonus3 bAddEff,Eff_Stun,200,ATF_SKILL; bonus bUnbreakableWeapon,1; },{},{}
 1675,Walking_Stick_,Gentleman's Staff,5,20,,500,40:125,,1,3,0x00818314,63,2,2,4,50,1,10,{ bonus bDex,1; },{},{}
 1676,Baculum_Daemonicum,Baculum Daemonicum,5,20,,1200,60:150,,1,2,0x00818315,63,2,2,4,80,1,10,{ bonus bAtkEle,Ele_Dark; bonus bUnbreakableWeapon,1; bonus bVariableCastrate,-10; if (BaseLevel > 99){ bonus bMatk,30; } if(getrefine() > 8){ bonus2 bResEff,Eff_Blind,3000; bonus bUseSPrate,-10; } },{},{}
-1677,Ru_Blue_Wand,Blue Wand,5,10,,0,50,,1,1,0x00000200,56,2,2,3,100,1,10,{ bonus bStr,5; bonus bInt,5; },{},{}
+1677,Ru_Blue_Wand,Blue Wand,5,10,,0,50,,1,1,0x00000200,56,2,2,3,100,1,10,{ bonus bDex,5; bonus bInt,5; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Missing 200 MATK

File: db/re/item_db.txt [hunk 5]
```diff
@@ -1165,6 +1165,7 @@
 1832,Velum_Claw,Vellum Claw,5,20,,650,160,,1,0,0x00008100,63,2,2,4,95,1,12,{ bonus2 bAddRace,RC_Player,100; .@r = getrefine(); if(.@r>7) { bonus2 bAddRace,RC_Player,20; } if(.@r>9) { bonus2 bVariableCastrate,"MO_EXTREMITYFIST",-100; bonus2 bFixedCastrate,"MO_EXTREMITYFIST",-100; bonus4 bAutoSpellOnSkill,"MO_EXPLOSIONSPIRITS","CH_SOULCOLLECT",1,10000; bonus2 bHPLossRate,500,3000; } },{},{}
 1833,Claw_Of_Flash,Claw of Flash,5,60000,,400,140,,1,2,0x00008100,63,2,2,4,105,1,12,{ bonus bAspd,1; bonus bAspdRate,getrefine(); bonus2 bAddSize,Size_Medium,getrefine(); },{},{}
 1834,TE_Woe_Fist,TE Woe Fist,5,0,,0,150,,1,0,0x00008100,63,2,2,3,40,1,12,{ bonus2 bAddRace,RC_Player,40; bonus2 bAddEff,Eff_Freeze,3000; },{},{}
+1835,Spartacus,Spartacus,5,0,,600,100,,1,0,0x00008100,63,2,2,4,50,2,12,{ .@r = getrefine(); bonus bAtkRate,.@r*1; if(.@r>9) { bonus bNoSizeFix,0; } },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Missing unbreakable bonus again.

File: db/re/item_db.txt [hunk 6]
```diff
@@ -1002,9 +1002,9 @@
 1650,P_Staff1,Eden Staff I,5,0,,0,60:125,,1,0,0x00818314,63,2,2,2,26,0,10,{ bonus bInt,2; },{},{}
 1651,P_Staff2,Eden Staff II,5,0,,0,60:150,,1,0,0x00818314,63,2,2,2,40,0,10,{ bonus bInt,3; },{},{}
 1652,Tourist_Staff,Tourist Staff,5,0,,500,35:0,,1,0,0x00818315,63,2,2,1,1,0,10,{ bonus bMatkRate,15; bonus bInt,2; bonus bAgi,1; },{},{}
-1653,Staff_Of_Healing_C,Staff of Healing,5,20,,0,10:100,,1,0,0x00008110,63,2,2,3,1,0,10,{ bonus bAtkEle,Ele_Holy; },{},{}
+1653,Staff_Of_Healing_C,Staff of Healing,5,20,,0,10:100,,1,0,0x00008110,63,2,2,3,1,0,10,{ bonus bAtkEle,Ele_Holy; bonus bHealPower,18; },{},{}
 1654,Mental_Stick,Mental Stick,5,20,,500,40:170,,1,1,0x00818315,63,2,2,3,102,1,10,{ .@r = getrefine(); if(.@r>5) { bonus2 bSkillAtk,"SO_PSYCHIC_WAVE",(.@r-5)*2; bonus bMaxHPrate,-(.@r-5)*2; } bonus2 bSkillVariableCast,"SO_PSYCHIC_WAVE",-3000; bonus2 bSkillUseSP,"SO_PSYCHIC_WAVE",-60; },{},{ itemheal 0,-100; }
-1655,Adventure_Staff,Adventure Staff,5,0,,0,60,,1,0,0x00810204,63,2,2,1,1,0,10,{ bonus bUnbreakableWeapon,1; },{},{}
+//1655,Adventure_Staff,Adventure Staff,5,0,,0,60,,1,0,0x00810204,63,2,2,1,1,0,10,{ bonus bUnbreakableWeapon,1; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: We need to check this again. I saw it was removed on DP, but this requires further investigations on our side.

File: db/re/item_db.txt (hunk 1 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Confirmed, I'll change. :-)

File: db/re/item_db.txt (hunk 2 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: @Lemongrass3110 In DivinePride says MaxHP -40 too, It just used _comma_ to not repeat. _(In my interpretation)_

But I had misunderstood too, I had understood so:

_Increase Aspd Rate in 1% every 10 base levels but decrease MaxHP in 40._

Then how would it look?

I've did it:

```
1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,30,1,1,6,{ bonus bMaxHP,200; bonus bAspdRate,5; bonus bUnbreakableWeapon,1; bonus bAspdRate,(-1*(BaseLevel/10)); bonus bMaxHP,(-40*(BaseLevel/10)); },{},{}
```

I didn't know if this is correct:

```
bonus bAspdRate,(-1*(BaseLevel/10));
```

File: db/re/item_db.txt (hunk 6 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Yes, it doesn't exist anymore.

File: db/re/item_db.txt (hunk 3 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Haha I was not sure.

So it should be **if>= 9 gives 5% and if>= 12 gives 12% _(5+7)_**, right?

File: db/re/item_db.txt (hunk 4 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Done. ✌️

**Pull request closed without merging**